lib/ramble/ramble/cmd/workspace.py

```diff
@@ -340,12 +340,21 @@ def workspace_remove(args):
 
 def workspace_concretize_setup_parser(subparser):
     """Concretize a workspace"""
-    pass
+    subparser.add_argument(
+        '-f', '--force_concretize',
```

Comment:
Can you change this from `--force_concretize` to `--force-concretize` ? That's how we have the arguments in most of the other commands, and I think it'd prefer to keep the underscore / hyphen issues consistent.

Reply:
Done